receiver/jaegerreceiver/jaeger_agent_test.go

```diff
@@ -50,10 +73,98 @@ func TestJaegerAgentUDP_ThriftBinary_6832(t *testing.T) {
 	})
 }
 
+func TestJaegerAgentUDP_ThriftBinary_PortInUse(t *testing.T) {
+	// This test confirms that the thrift binary port is opened correctly.  This is all we can test at the moment.  See above.
```

Comment:
Q.: did you check the current status of the above? Is it still an issue?

Reply:
I removed the `t.Skipf()` from the above test and it still failed.  However, I'm wondering if the jaeger go client that's used in the test will only work for thrift compact.  The docs suggest that only the node jaeger client uses thrift binary on 6832:

https://www.jaegertracing.io/docs/1.15/deployment/#agent

I haven't really dug deeper than this however.

Reply:
Thanks for checking it.